tests/app/clients/test_aws_sns.py

```diff
@@ -51,3 +52,27 @@ def test_send_sms_with_long_code_successful_returns_aws_sns_response(notify_api,
             'AWS.MM.SMS.OriginationNumber': {'DataType': 'String', 'StringValue': sender},
         }
     )
+
+
+def test_send_sms_to_us_number_successful_returns_aws_sns_response(notify_api, mocker):
```

Comment:
Could you add an additional test demonstrating that if we pass a custom sender, the US toll-free number is used for US phone numbers and not the passed sender?

Thanks!

Reply:
Added test case!